01_EDA.ipynb

```diff
@@ -0,0 +1,1752 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "f1.hist(customers_info, 'Gender')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that both gender are equally distributed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## C. Age"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's calculte the date from the date of birth to the date of transaction to see age distribution when they buy theirs items"
```

Comment:
Let's calculate the date from the date of birth to the date of transaction to see age distribution when they buy their items